src/main/java/com/rackspace/salus/event/ingest/services/IngestService.java

```diff
@@ -52,18 +54,23 @@
   private final KapacitorConnectionPool kapacitorConnectionPool;
   private final ConcurrentHashMap<EngineInstance, InfluxDB> influxConnections =
       new ConcurrentHashMap<>();
-  private static final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ISO_INSTANT;
+  private static final DateTimeFormatter xdateTimeFormatter = DateTimeFormatter.ISO_INSTANT;
+  private final Counter metricsConsumed;
+  private final Counter metricsWrittenToKafka;
 
   @Autowired
   public IngestService(KafkaTopicProperties kafkaTopicProperties,
                        EventIngestProperties eventIngestProperties,
                        EventEnginePicker eventEnginePicker,
-                       KapacitorConnectionPool kapacitorConnectionPool) {
+                       KapacitorConnectionPool kapacitorConnectionPool,
+                       MeterRegistry meterRegistry) {
     this.kafkaTopicProperties = kafkaTopicProperties;
     this.eventIngestProperties = eventIngestProperties;
     this.eventEnginePicker = eventEnginePicker;
     this.kapacitorConnectionPool = kapacitorConnectionPool;
-  }
+    metricsConsumed = meterRegistry.counter("ingest","operation", "consume");
+    metricsWrittenToKafka = meterRegistry.counter("ingest","operation", "written");
```

Comment:
I think the variable name should be this since this service writes to the other "K" backend:
```suggestion
    metricsWrittenToKapacitor = meterRegistry.counter("ingest","operation", "written");
```

Reply:
much better, thanks!